.gitignore

```diff
@@ -1,2 +1,13 @@
 # emacs
-*~
\ No newline at end of file
+*~
+
+# pycharm
+.idea/
+
+.ipynb_checkpoints/
+limnos.egg-info/
+limnos/.ipynb_checkpoints/
+limnos/Untitled.ipynb
+limnos/__pycache__/
+limnos/tests/__pycache__/
```

Comment:
It is sufficient to have a single `__pycache__` line; we don't need one for each subfolder.